content/tbb/tbb-14/contents.lr

```diff
@@ -6,4 +6,17 @@ seo_slug: installing-add-on-extensions-tor-browser
 ---
 description:
 It's strongly discouraged to install new add-ons in Tor Browser, because they can compromise your privacy and security.
+
+How?
+Basically, each browser's settings and features create what is called a "browser fingerprint".
+Most browsers inadvertently create a unique fingerprint for each user which can be tracked across the internet.
+Tor Browser is specifically engineered to have a nearly identifcal (we're not perfect!) fingerprint across it's users.
+This means each Tor Browser user looks like every other Tor Browser user, making it difficult to track any individual user.
+
+Now, how does this relate to installing new add-ons in Tor Browser?
+Installing new add-ons may affect Tor Browser in unforeseen ways and potentially make your Tor Browser fingerprint unique.
+If your copy of Tor Browser has a unique fingerprint, your browsing activities can be deanonymized and tracked even though you are using Tor Browser.
+
 Tor Browser already comes installed with two add-ons — <mark><a href="https://www.eff.org/https-everywhere">HTTPS Everywhere</a></mark> and <mark><a href="https://noscript.net/">NoScript</a></mark> — and adding anything else could deanonymize you.
+
```

Comment:
Fingerprinting is a great point! 

There's also another points that we could incorporate:

Malicious add-ons: a random add-on installed could compromise your anonymity. Or even non-malicious, they could change your Security settings (NoScript), for example.

Attack surface. Installing new add-ons can increase ways to attack your Tor Browser as the new add-on could leak private information about your OS, for example.

Reply:
Good point! I forgot about those attack scenarios.